pkg/check/auth/auth.go

```diff
@@ -58,6 +58,9 @@ func (c *Check) Run(ctx context.Context, cluster *bee.Cluster, opts interface{})
 	for _, node := range restricted.Nodes() {
 		client := node.Client()
 
+		fmt.Println("trying to authorize with", o.Role, o.AdminUsername, o.AdminPassword)
```

Comment:
You should replace "authorize" with "authenticate" since we're doing authentication. Also on line 66 below.